quantum/encoder.c

```diff
@@ -68,3 +70,23 @@ void encoder_read(void) {
     encoder_value[i] %= ENCODER_RESOLUTION;
   }
 }
+
+#ifdef SPLIT_KEYBOARD
+void encoder_value_raw(int8_t* sencoder) {
+  memcpy((void*)sencoder, (const void *)encoder_value, sizeof(encoder_value));
+}
+
+void encoder_update_raw(int8_t* sencoder) {
+  for (int i = 0; i < NUMBER_OF_ENCODERS; i++) {
+    if (encoder_value[i] != sencoder[i]) {
+      encoder_value[i] = sencoder[i];
+      if (encoder_value[i] >= ENCODER_RESOLUTION) {
+        encoder_update_kb(i, COUNTRECLOCKWISE);
```

Comment:
Typo. Counter

Reply:
It's that way in the existing encoder code, so technically it's correct here. Still, it should be fixed.

Reply:
ya, the fact that it is also a define for a true / false bool is also silly, will fix it.

Reply:
fixed